pkgs/top-level/all-packages.nix

```diff
@@ -9883,6 +9883,8 @@ with pkgs;
 
   textadept11 = callPackage ../applications/editors/textadept/11 { };
 
+  textpieces = callPackage ../applications/misc/textpieces { };
```

Comment:
```suggestion
textpieces = callPackage ../applications/misc/textpieces { };
```

Reply:
Is the suggestion here to remove the indentation intentional?

Reply:
Oh, the github mobile app must have broken formatting, or I was just blind.